psycopg_c/build_backend/psycopg_build_ext.py

```diff
@@ -9,13 +9,16 @@
 
 import os
 import sys
+import shutil
 import subprocess as sp
 from distutils import log
 from distutils.command.build_ext import build_ext
 
 
 def get_config(what: str) -> str:
     pg_config = "pg_config"
+    if sys.platform == "win32":
+        pg_config = shutil.which("pg_config")
```

Comment:
Not sure why `subprocess.run` will fail, but this should fix it without any unexpected side effect.

Reply:
Yes, it the libpq version logged in tests is expected version

Reply:
The error I want to fix is that now subprocess throw a error that it can't find pg_config, even it's already in the path

Reply:
Maybe you are setting the path wrongly? Elsewhere in this workflow file we use something like:

```
echo "PATH=/opt/homebrew/opt/postgresql@${PG_VERSION}/bin:$PATH" >> $GITHUB_ENV
```

and we don't use `$GITHUB_PATH`.

Can you print the `$PATH` in the place where it is used? Can you try to use a technique as the above? (remember that the path separator on windows is `;`, not `:`).

Reply:
I'm pretty sure the path is correct, because we just call `pg_config.exe` after it's installed

![image](https://github.com/user-attachments/assets/c1ca030b-3bbc-4304-9877-f7c9033becb7)

Reply:
also `echo ... > $GITHUB_ENV` has same effect with `echo ... > $GITHUB_PATH`

Reply:
Looks like we do not need it now, don't know why, didn't want to dig deep into it 😅